specs/core/1_shard-data-chains.md

```diff
@@ -270,7 +270,8 @@ def get_shard_block_proposer_index(state: BeaconState,
         return None
 
     MAX_RANDOM_BYTE = 2**8 - 1
-    seed = hash(get_seed(state, current_epoch) + int_to_bytes(shard, length=8) + int_to_bytes(slot, length=8))
+    seed = get_seed(state, current_epoch, DOMAIN_SHARD_PROPOSER)
```

Comment:
Suggest calling this `epoch_seed` or something to distinguish it from the next assignment.
```suggestion
    epoch_seed = get_seed(state, current_epoch, DOMAIN_SHARD_PROPOSER)
```

An alternative would be something like:

```python
seed = hash(get_seed(state, current_epoch, DOMAIN_SHARD_PROPOSER)
            + int_to_bytes(shard, length=8)
            + int_to_bytes(slot, length=8))

Reply:
Went with `epoch_seed`. This will all get cleaned up shortly anyway with #1383.